src/collect.rs

```diff
@@ -207,37 +213,8 @@ impl KeyCollection {
                 vec_add(&mut beta_share, y_p1);
 
                 let flp_proof_share = &self.all_flp_proof_shares[client_index];
```

Comment:
Reviwer note: this file is easier to review with: "hide whitespace" **disabled**. The main changie is moving the joint randomness computation into its own function.